tests/nosetests/pyanaconda_tests/__init__.py

```diff
@@ -101,8 +117,9 @@ def check_kickstart_interface(test, interface, ks_in, ks_out=None, ks_valid=True
         return result
 
     # Generate a kickstart
-    ks_out = dedent(ks_out).strip()
-    test.assertEqual(ks_out, interface.GenerateKickstart().strip())
+    ks_out = clear_version_from_kickstart_string(dedent(ks_out)).strip()
```

Comment:
`ks_out` is the expected kickstart string, so it shouldn't contain the `# Generated` comment. Could you try to run the tests without modifying `ks_out`?

Reply:
Ha, you're right, I got confused by the reversed order at the other place I guess...